NAS-103229 doc updated task manager (#1567)
freenas/freenas-docs: userguide/taskmanager.rst

Task Manager
============

The task manager shows a list of tasks performed by the %brand% system

> in descending order. The start time, progress, finish time, and whether

Reviewer: ```suggestion
starting with the most recent. Click a task name to show start time, progress, finish time, and whether
```

> the task succeeded or failed are also displayed. If a task failed, the

Reviewer: ```suggestion
the task succeeded are displayed. If a task failed, the
```

error status is shown.

> Some tasks have a log. If a task run has a long, select that task and

Reviewer: ```suggestion
Tasks with log file output have a :guilabel:`View Logs` button to show the log files.
```

> and click :guilabel:`View Logs`.

Reviewer: ```suggestion
```
